src/main/java/com/amihaiemil/docker/Plugins.java

```diff
@@ -26,39 +26,42 @@
 package com.amihaiemil.docker;
 
 import java.io.IOException;
+import javax.json.JsonArray;
 
 /**
  * Plugins API.
  * @author Boris Kuzmic (boris.kuzmic@gmail.com)
  * @since 0.0.7
- * @todo #208:30min Continue implementing Plugins API methods. More information
- *  about available methods:
- *  https://docs.docker.com/engine/api/v1.35/#tag/Plugin
+ * @todo #232:30min Implement getting plugin privilages. More information
+ *  about API method can be found at:
+ *  https://docs.docker.com/engine/api/v1.35/#operation/GetPluginPrivileges
  */
-public interface Plugins {
+public interface Plugins extends Iterable<Plugin> {
 
     /**
      * Create a plugin.
      * @param name Name of the plugin.
+     * @param pluginDataDir Path to plugin's data dir.
      * @throws IOException If something goes wrong.
      * @throws UnexpectedResponseException If the status response is not
      *  the expected one (200 OK).
      * @see <a href="https://docs.docker.com/engine/api/v1.35/#operation/PluginCreate">Create a plugin</a>
      */
-    void create(final String name)
+    void create(final String name, final String pluginDataDir)
```

Comment:
@bkuzmic We do not need to prefix this parameter with `plugin`, just using `dataDir` or `directory` would be fine.